### PR TITLE
Pfe-tabs - add centered alignment

### PR DESCRIPTION
**Pull request by starryeyez024**
Adds an attribute which centers the pfe-tabs, if the orientation is horizontal.

### CODE REVIEW

File: elements/pfe-tabs/demo/index.html [hunk 1]
```diff
@@ -101,8 +101,8 @@ <h2>Content 3</h2>
       </pfe-tabs>
     </section>
     <section class="example">
-      <h2>Style: Horizontal wind</h2>
-      <pfe-tabs pfe-variant="wind">
+      <h2>Style: Horizontal wind with center alignment</h2>
+      <pfe-tabs pfe-align="center" pfe-variant="wind">
```
Comment (castastrophe) [inline elements/pfe-tabs/demo/index.html]: What would you think of the attribute name `pfe-tab-align` instead?  To clarify that the alignment relates to the tabs and not the content for example?

File: elements/pfe-tabs/demo/index.html [hunk 2]
```diff
@@ -173,8 +173,8 @@ <h2>Content 3</h2>
       </pfe-tabs>
     </section>
     <section class="example">
-      <h2>Style: Vertical earth</h2>
-      <pfe-tabs vertical pfe-variant="earth">
+      <h2>Style: Vertical earth </h2>
+      <pfe-tabs pfe-align="center" >
```
Comment (castastrophe) [inline elements/pfe-tabs/demo/index.html]: Noting this from the demo - I think you mentioned this one needed to have the `vertical` attribute added back.

File: elements/pfe-tabs/demo/index.html (hunk 1 above)
Comment (castastrophe) [reply elements/pfe-tabs/demo/index.html]: @starryeyez024 bump ^

File: elements/pfe-tabs/demo/index.html (hunk 2 above)
Comment (castastrophe) [reply elements/pfe-tabs/demo/index.html]: @starryeyez024 bump ^

File: elements/pfe-tabs/demo/index.html (hunk 1 above)
Comment (starryeyez024) [reply elements/pfe-tabs/demo/index.html]: works for me :)

File: elements/pfe-tabs/demo/index.html [hunk 3]
```diff
@@ -173,8 +173,8 @@ <h2>Content 3</h2>
       </pfe-tabs>
     </section>
     <section class="example">
-      <h2>Style: Vertical earth</h2>
-      <pfe-tabs vertical pfe-variant="earth">
+      <h2>Style: Vertical earth </h2>
+      <pfe-tabs vertical pfe-align="center" >
```
Comment (castastrophe) [inline elements/pfe-tabs/demo/index.html]: Does this need the earth variant added back to it?

File: elements/pfe-tabs/demo/index.html (hunk 1 above)
Comment (starryeyez024) [reply elements/pfe-tabs/demo/index.html]: updated

File: elements/pfe-tabs/demo/index.html (hunk 3 above)
Comment (starryeyez024) [reply elements/pfe-tabs/demo/index.html]: gah, thank you for catching that. Twas unintentional, and has been resurrected.

**Pull request merged**